coral/src/app/features/topics/schema-request/TopicSchemaRequest.test.tsx

```diff
@@ -292,24 +292,23 @@ describe("TopicSchemaRequest", () => {
       cleanup();
     });
 
-    // The new placeholder behaviour is to select the first option when no placeholder is provided
-    // So this situation will never happen, as there always be a value selected in this context
-    // @TODO https://github.com/aiven/klaw/issues/495
-    // it("shows error when user does not fill out environment select", async () => {
-    //   const form = getForm();
-    //   const select = within(form).getByRole("combobox", {
-    //     name: /Select environment/i,
-    //   });
-
-    //   select.focus();
-    //   await userEvent.keyboard("{ArrowDown}");
-    //   await userEvent.keyboard("{ESC}");
-    //   await userEvent.tab();
-
-    //   const error = await screen.findByText("The environment is required.");
-    //   expect(error).toBeVisible();
-    //   expect(select).toBeInvalid();
-    // });
+    it("shows error when user does not fill out environment select", async () => {
```

Comment:
This is a very special behaviour testcase for `TopicSchemaRequest.test.tsx`. I probably would place this in `Form.test.tsx` and in the components using this, would implicitly expect this to work. However, there is not harm having this here, more is more!

Reply:
Good point! Form already has a [test](https://github.com/aiven/klaw/blob/511d2925a9195a202f22634aeec21b88a931653d/coral/src/app/components/Form.test.tsx#L247) that kind of checks for this, but I would add an additional one with a more common schema (not an enum) to explicit asserting that there is an error shown if a required select does not have a value. 

And then the test is not really needed here again, as long as there is a confirmation that the select in this component is required.

Reply:
Updated the test cases in `Form`. 

But since in cases of select an error for a required field is only shown correctly based on the schema that is used in the component, I would leave this test. It makes sure the select is correctly marked as required, the schema is correct and the right error message is shown!